srcpkgs/wayclip/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'wayclip'
+pkgname=wayclip
+version=0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="wayland-devel"
```

Comment:
```suggestion
makedepends="wayland-devel"
```

Reply:
Thanks

Reply:
it needs to be in both `hostmakedepends` and `makedepends` (it runs `wayland-scanner`)

Reply:
@classabbyamp, got it, thanks :D